docs/workforce-management-framework/topics/con-workflow-step.adoc

```diff
@@ -53,12 +62,15 @@ module.exports = {
 };
 ----
 
-== Development 
-When developing a custom Step package, it is recommend that you clone the link:https://github.com/feedhenry-raincatcher/step-template.git[step-template] repository and 
-modify it. Alternatively you can use link:https://www.npmjs.com/package/generator-rcstep[generator-rcstep] npm package to create a step. 
-See the link:https://github.com/feedhenry-raincatcher/step-template.git[README.md] for furhter information on modifying a step and importing it into 
-the portal and mobile applications.
-
+== Development
+ifdef::downstream[]
+When developing a custom _Step Package_, Red Hat recommends that you clone the link:https://github.com/feedhenry-raincatcher/step-template.git[step-template] repository and modify it.
```

Comment:
I'm against doing that path here and splitting between upstream/downstream.

We can just say that "Project provides template for steps that is located here:" 
I will avoid any recommendations etc. 

That's really important to me to avoiding hassle between maintaining upstream and downstream versions.